sites/partners/__mocks__/tabbable.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-env jest */
```

Comment:
Can you walk me through why we have this?

Reply:
The way the Seeds Drawer/Dialog handles focus traps messed up JSDom tests (it erroneously thought the currently focused element wasn't visible on-screen), so this is the workaround.